src/event.rs

```diff
@@ -998,9 +1002,9 @@ bitflags! {
         // left and right modifiers are currently commented out, but we should be able to support
         // them in a future release
         /// The "shift" key.
-        const SHIFT = 0b100 << 0;
-        // const LSHIFT = 0b010 << 0;
-        // const RSHIFT = 0b001 << 0;
+        const SHIFT = 0b100;
+        // const LSHIFT = 0b010;
+        // const RSHIFT = 0b001;
```

Comment:
These are the kinds of warnings I don't like about clippy; the bitshift was obviously there to keep the pattern consistent. But don't get me wrong, I think this change is OK. I just wanted to get this off my chest :D

Reply:
Yep, same here.